docs/reference/deploy/master-config-reference.rst

```diff
@@ -1538,6 +1538,41 @@ Security-related configuration settings.
       certificate is not signed by a well-known CA; cannot be specified if ``skip_verify`` is
       enabled.
 
+***********************
+ ``logging_retention``
+***********************
+
+Specifies configuration settings for the logging retention of trial logs.
+
+``days``
+========
+
+Number of days to retain logs for by default. This can be overridden on a per-experiment basis in
+the :ref:`experiment configuration <log-retention-days>`. Values should be between ``-1`` and
+``32767``. The default value is ``-1`` and will retain logs indefinitely. If set to ``0``, logs will
+be deleted during the next cleanup.
```

Comment:
``days``
========

Number of days to retain logs for by default. This can be overridden on a per-experiment basis in
the :ref:`experiment configuration <log-retention-days>`. Values should be between ``-1`` and
``32767``. The default value is ``-1``, retaining logs indefinitely. If set to ``0``, logs will
be deleted during the next cleanup.